query-algebrizer/src/cc.rs

```diff
@@ -67,16 +67,30 @@ pub enum DatomsColumn {
     ValueTypeTag,
 }
 
+impl DatomsColumn {
+    pub fn as_str(&self) -> &'static str {
+        use DatomsColumn::*;
+        match *self {
+            Entity => "e",
```

Comment:
It would be nice if we could eventually move these concrete details into a shared location, which I've been thinking of as the `db` crate (once the `tx` stuff is moved out of it).  These are the kinds of things an implementation backed by different storage would want to change.